docs/extending/application-actions.md

```diff
@@ -124,4 +124,5 @@ Below is the list of public actions types you can use in the plugin definitions
 | 1.8.0   | VIEW_NODE              | NodeId<`string`> , [ViewNodeExtras](../features/file-viewer.md#details)<`any`> | Lightweight preview of a node by id. Can be invoked from extensions. For details also see [File Viewer](../features/file-viewer.md#details) |
 | 1.8.0   | CLOSE_PREVIEW          | n/a                                                                            | Closes the viewer ( preview of the item )                                                                                                   |
 | 1.9.0   | RESET_SELECTION        | n/a                                                                            | Resets active document list selection                                                                                                       |
+| 1.10.0  | CONTEXT_MENU              | MouseEvent                                                  | Invoke context menu for [DocumentListComponent](https://www.alfresco.com/abn/adf/docs/content-services/components/document-list.component) |
 | 2.0.0   | CREATE_FILE_FROM_TEMPLATE        | n/a | Invoke a dialog listing `Node Templates` folder. Selected template can be copied in the current folder from where tha action was invoked                                                                                               |
```

Comment:
this will be fixed by https://github.com/Alfresco/alfresco-content-app/pull/1299